deploy/src/main/deploy/helm/templates/artemis/artemis-deployment.yaml

```diff
@@ -27,12 +27,17 @@ spec:
     spec:
       containers:
       - command:
-        - /opt/artemis/bin/artemis
-        - run
-        - xml:/etc/hono/artemis-bootstrap.xml
+        - /opt/apache-artemis/bin/launch.sh
+        - start
         env:
-        - name: ARTEMIS_CONFIGURATION
-          value: /etc/hono
+        - name: AMQ_NAME
+          value: custom
+        - name: HOME
+          value: /var/run/artemis/
+        - name: JAVA_INITIAL_MEM_RATIO
+          value: "30"
+        - name: JAVA_MAX_MEM_RATIO
+          value: "70"
```

Comment:
Does this mean that we expect Artemis to successfully start up with 30% of 256MB?

Reply:
It's a bit confusing, as it is 30% of max value (360M) which is roughly what it was configured before. BTW. Defaults are 256M max and 64M init, which works fine in system tests.

When we switch to openjdk-11 profile we can switch to -XX:InitialRAMPercentage / -XX:MaxRAMPercentage config combo.

Reply:
Ok, so currently, the Artemis image uses Java < 9 and manually computes the Xms and Xmx values from the environment variables `JAVA_INITIAL_MEM_RATIO` and `JAVA_MAX_MEM_RATIO`?

Reply:
It uses JDK 11, but launch scripts use "legacy" mode which compute manually all  the parameters. I'll need to investigate further to see what does it take to upgrade this image to use proper mode.

Reply:
I just wanted to understand. If you think that, for the time being, using it the way you did is the right way, then I am fine with that ...

Reply:
Sure thing. Yeah, I think this is good to go and we should continue to work on getting artemis image to the latest features. Do you want me to squash these commits?

Reply:
IMHO this small change doesn't require more than one commit, does it?